guides/v15-mdc-migration.md

```diff
@@ -501,10 +501,46 @@ DOM and CSS of the components, you may need to tweak some of your application's
 * The label is closer to the enabled toggle
 
 ### Slider
-<!-- TODO(wagnermaciel): review -->
 
 * Sliders now work with mobile device screen readers.
 
+* The slider template API has changed from a single `<mat-slider>` element to a `<mat-slider>`
+  element which contains one or two `<input>` elements (depending on whether the slider should)
+  be a standard or range slider. E.g.
+  ```html
+    <!-- Single slider -->
+    <mat-slider>
+      <input matSliderThumb>
+    </mat-slider>
+
+    <!-- Range slider -->
+    <mat-slider>
+      <input matSliderStartThumb>
+      <input matSliderEndThumb>
+    </mat-slider>
+  ```
+
+* Range sliders are now supported. A range slider uses two native slider inputs to control two
+  multiple values.
+
+* The new `discrete` property on the `<mat-slider>` now controls whether the slider has tick marks
+  and a value indicator tooltip. It replaces `thumbLabel`.
+  
+* The `tickInterval` property has been replaced with the boolean `showTickMarks` and `step`.
+
+* The `displayValue` property has been replaced by `displayWith`.
```

Comment:
does `displayWith` work exactly the same way `displayValue` did? If so maybe say "renamed to" instead of "replaced by"